config/samples/storage_v1_csm_powerscale.yaml

```diff
@@ -294,6 +294,7 @@ spec:
         image: dellemc/dell-replication-controller:v1.4.0
         envs:
           # TARGET_CLUSTERS_IDS: comma separated list of clusters id of the targets clusters. DO NOT include the source(wherever CSM Operator is deployed) cluster id
```

Comment:
id is normally in uppercase, i.e. ID. You don't have to change but just FYI.

Reply:
Done